packages/core/docs/guide/user-profile.md

```diff
@@ -0,0 +1,203 @@
+
+Like with all other composables, it's important to remember to call `load` before accessing any other property or function of `useUser`. Otherwise, `isAuthenticated` will always return `false`.
+
+```js{8,18}
+import { useUser } from '{INTEGRATION}';
+import { onSSR } from '@vue-storefront/core';
+
+export default {
+  setup () {
+    const {
+      load,
+      isAuthenticated
+    } = useUser();
+
+    onSSR(async () => {
+      await load();
+    });
+
+    return {
+      load,
+      isAuthenticated
+    }
+  }
+}
+```
+
+## Loading current user
+
+When you confirmed that the user is logged in, you may safely access user data.
+
+```js{8,18}
+import { useUser } from '{INTEGRATION}';
+import { onSSR } from '@vue-storefront/core';
+
+export default {
+  setup () {
+    const {
+      load,
+      user
+    } = useUser();
+
+    onSSR(async () => {
+      await load();
+    });
+
+    return {
+      load,
+      user
+    }
+  }
+}
+```
+
+Similar to other composables, it's important to remember to not directly access `user` properties, but use `userGetters`. This will ensure that your code is integration-agnostic and proof to (some) future changes.
+
+```js{1,3,17,22}
+import { useUser, userGetters } from '{INTEGRATION}';
+import { onSSR } from '@vue-storefront/core';
+import { computed } from '@vue/composition-api';
+
+export default {
+  setup () {
+    const {
+      load,
+      user
+    } = useUser();
+
+    onSSR(async () => {
+      await load();
+    });
+
+    const firstName = computed(() => userGetters.getFirstName(user.value));
+
+    return {
+      load,
+      user,
+      firstName
+    }
+  }
+}
+```
+
+## Modifying user
+
+Updating user data (except for the current password, which is described in the [Changing password](#changing-password) section) can be done using `updateUser` method in `useUser` composable.
+
+```js
+import { useUser } from '{INTEGRATION}';
+
+export default {
+  setup () {
+    const { updateUser } = useUser();
+
+    const onSubmit = async (formData) => {
+      try {
+        await updateUser({ user: formData });
+        // 'user' property is now updated
+      } catch (error) {
+        // handle update error
+      }
+    };
+
+    return {
+      onSubmit
+    }
+  }
+}
+```
+
+## Changing password
+
+Updating user password can be done using `changePassword` method in `useUser` composable. It requires the current and new password, to confirm user identity.
+
+```js
+import { useUser } from '{INTEGRATION}';
+
+export default {
+  setup () {
+    const { changePassword } = useUser();
+
+    const onSubmit = async (formData) => {
+      try {
+        await changePassword({
+          current: formData.currentPassword,
+          new: formData.newPassword
+        });
+      } catch (error) {
+        // handle update error
+      }
+    };
+
+    return {
+      onSubmit
+    }
+  }
+}
+```
+
+## Managing addresses (billing and shipping)
```

Comment:
The linked documents will not be a part of this docs as stated in ToC (they will be parts of API Reference). We're rewriting it to more human-readable version. Please include those explanations as well.

Reply:
Done